pom.xml

```diff
@@ -23,12 +23,15 @@
         </developer>
     </developers>
     <scm>
-        <connection>scm:git:https://github.com/jenkinsci/clover-plugin.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/jenkinsci/clover-plugin.git</developerConnection>
-        <url>https://github.com/jenkinsci/clover-plugin</url>
-        <tag>clover-4.7.0</tag>
+        <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/${gitHubRepo}.git</developerConnection>
+        <url>https://github.com/${gitHubRepo}</url>
+        <tag>${scmTag}</tag>
```

Comment:
Where's the default value for the `scmTag`?

Reply:
When I run `mvn help:effective-pom` it shows the value of `scmTag` as `HEAD`.

I am confident that the syntax works as expected for `mvn release:prepare release:perform` because that is the syntax used in the ansicolor plugin, the git plugin, the git client plugin, multiple Pipeline plugins, and in Jenkins core.  Unfortunately, I'm not enough of an expert in Apache Maven to describe why it works, only that it works for both `mvn release:prepare release:perform` and for continuous delivery of Jenkins plugins through JEP-229 (as is done with the schedule build plugin, the structs plugin, and many others) .